<launch>

    <arg name="robot_name" default="alpha"/>
    <arg name="model_num"  default="1"/>
    <arg name="is_sim" default="false"/>
    <node pkg ="tensorflow_dvl" name = "tensorflow_dvl" type = "tf_dvl_node.py" output="screen" >
        <param name="/u_model_dir" type="string" value="$(find tensorflow_dvl)/config/models/u/model$(arg model_num)/"/>
        <param name="/v_model_dir" type="string" value="$(find tensorflow_dvl)/config/models/v/model$(arg model_num)/"/>
        <param name="/w_model_dir" type="string" value="$(find tensorflow_dvl)/config/models/w/model$(arg model_num)/"/>
        <param name="/hz" type="double" value="5"/>
        <param name="/enable_depth" type="double" value="0.2"/>
    </node>

</launch>